### PR TITLE
Fix Linux PPC wheel build yml script

### PR DESCRIPTION
**Pull request by chaeyeunpark**
There is a typo in the PPC wheel build yml script. This PR fixes it.

### CODE REVIEW

File: .github/workflows/wheel_linux_ppc64le.yml [hunk 1]
```diff
@@ -44,7 +45,7 @@ jobs:
     outputs:
       matrix: ${{ steps.pyver.outputs.matrix }}
 
-linux-wheels-ppc64le:
+  linux-wheels-ppc64le:
```
Comment (mlxd) [inline .github/workflows/wheel_linux_ppc64le.yml:48]: Ooops, well spotted!

**Review by mlxd — COMMENTED** [review]
Well spotted @chaeyeunpark !
All good to merge once the builds are fine and the on pull-request is removed.

**Review by trevor-vincent — APPROVED** [review]
Nice catch! Looks good.

File: .github/workflows/wheel_linux_ppc64le.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ name: Wheel::Linux::PowerPC
 # **Who does it impact**: Wheels to be uploaded to PyPI.
 
 on:
+  pull_request: # remove before merge
```
Comment (mlxd) [inline .github/workflows/wheel_linux_ppc64le.yml:9]:
```suggestion
```

Nice work. I'll approve and mark this to be committed directly

**Review by mlxd — APPROVED** [review]

**Pull request merged**